tensorboard/webapp/reloader/reloader_component_test.ts

```diff
@@ -161,4 +182,158 @@ describe('reloader_component', () => {
 
     fixture.destroy();
   }));
+
+  it('does not reload if document is not visible', fakeAsync(() => {
+    store.setState(
+      createState(
+        createCoreState({
+          reloadPeriodInMs: 5,
+          reloadEnabled: true,
+        })
+      )
+    );
+    const fixture = TestBed.createComponent(ReloaderComponent);
+    fixture.detectChanges();
+
+    tick(5);
+    expect(dispatchSpy).toHaveBeenCalledTimes(1);
+
+    simulateVisibilityChange(false);
+    tick(5);
+    expect(dispatchSpy).toHaveBeenCalledTimes(1);
+
+    fixture.destroy();
```

Comment:
do this in `afterEach`?

Reply:
`fixture` is scoped to the test function. Is it worth instead scoping it more broadly?

Reply:
feel free to hoist it if you find it tedious, too. Else, ignore this :)

Reply:
I attempted to do this. I agree it's tedious. Unfortunately fakeAsync() will complain that "1 timer(s) still in the queue." if fixture isn't destroyed before the end of the test (because the destroy clears a Timeout). I could not find any documentation explaining how to make this work with fixture.destroy() in afterEach().